docs/cmf_client/step-by-step.md

```diff
@@ -1,35 +1,30 @@
-# Getting started with cmf
+# Quick start with cmf-client
 Common metadata framework (cmf) has the following components:
 
 - **Metadata Library** exposes API’s to track the pipeline metadata. It also provides API’s to query the stored metadata. 
 - **cmf-client** interacts with the server to pull or push metadata from or to the cmf-server.
 - **cmf-server** interacts with all the remote clients and is responsible to merge the metadata transferred by the cmf-client and manage the consolidated metadata. 
 - **Central Artifact Repositories** hosts the code and data. 
 
+## Install cmf library i.e. cmflib
+Before proceeding, ensure that the CMF library is installed on your system. If not, follow the installation instructions provided inside the [CMF in a nutshell](../index.md) page. 
+
+## Install cmf-server
+cmf-server is a key interface for the user to explore and track their ML training runs. It allows users to store the metadata file on the cmf-server. The user can retrieve the saved metadata file and can view the content of the saved metadata file using the UI provided by the cmf-server.
+
+Follow the instructions on the [Getting started with cmf-server](../cmf_server/cmf-server.md) page for details on how to setup a cmf-server.
+
 ## Setup a cmf-client 
 cmf-client is a tool that facilitates metadata collaboration between different teams or two team members. It allows users to pull or push metadata from or to the cmf-server.
 
 Follow the below-mentioned steps for the end-to-end setup of cmf-client:-
 
-**Pre-Requisites**
-
-- Python 3.8+
```

Comment:
Why did we get rid of python version ?

Reply:
We have reorganized the pages. On the main page (i.e., 'CMF in a Nutshell'), we have included all the installation steps for the CMF library, as shown in the image below. 
![Screenshot from 2024-05-10 11-54-31](https://github.com/HewlettPackard/cmf/assets/167299982/023317ac-f49f-4d9b-a736-561e432db93b)




In the 'Getting Started with the CMF Client' page, we have specified that the CMF server and client should be installed after configuring the CMF library, for which installation steps are provided on the main page.
![Screenshot from 2024-05-10 11-56-34](https://github.com/HewlettPackard/cmf/assets/167299982/be819d60-ff01-46fb-a25a-ec3ad78c5e35)